src/styles.css

```diff
@@ -32,3 +32,23 @@ body,
 
 .footer {
 }
+
+/* styles Mailchimp form button*/
+button {
+  background-color: #1976d2;
+  color: white;
+  border-radius: 10px;
+  border: none;
+  padding: 8px;
+  margin: 5px 0;
+  font-size: 0.9em;
+}
+
+/* styles Mailchimp form input */
+input {
+  padding: 7px;
+  margin: 0 5px 0 0;
+  border-radius: 10px;
+  border: 1px black solid;
+  font-size: 0.9em;
+}
```

Comment:
This will style buttons/inputs globally. Instead you should look into adding this in some sort of MailChimp wrapper.

Reply:
removed from styles.css and put in component